lib/actions/app.coffee

```diff
@@ -146,11 +146,15 @@ exports.associate =
 
 		This command adds a 'resin' git remote to the directory and runs git init if necessary.
 
+		Notice this command asks for confirmation interactively.
+		You can avoid this by passing the `--yes` boolean option.
```

Comment:
@Page- It's an alias to `--yes`, and will be displayed below automatically when doing `resin help app associate`.